### PR TITLE
Added Apcupsd

### PR DESCRIPTION
**Pull request by angrycuban13**
# Pull request

**Purpose**
Adding a new app to DockSTARTer

**Open Questions and Pre-Merge TODOs**
Check all boxes as they are completed

- [x] Use github checklists. When solved, check the box and explain the answer.

**Learning**
Describe the research stage
Links to blog posts, patterns, libraries or addons used to solve this problem

**Requirements**
Check all boxes as they are completed

- [x] These changes meet the standards for [contributing](https://github.com/GhostWriters/DockSTARTer/blob/master/.github/CONTRIBUTING.md).
- [x] I have read the [code of conduct](https://github.com/GhostWriters/DockSTARTer/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: compose/.apps/apcupsd/apcupsd.labels.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+      com.dockstarter.appvars.apcupsd_enabled: "false"
+      com.dockstarter.appvars.apcupsd_debug: ""
+      com.dockstarter.appvars.apcupsd_host: ""
+      com.dockstarter.appvars.apcupsd_infuxdb_database: ""
+      com.dockstarter.appvars.apcupsd_infuxdb_password: ""
+      com.dockstarter.appvars.apcupsd_infuxdb_user: ""
+      com.dockstarter.appvars.apcupsd_interval: ""
+      com.dockstarter.appvars.apcupsd_network_mode: ""
+      com.dockstarter.appvars.infuxdb_host: ""
+      com.dockstarter.appvars.infuxdb_port: ""
```
Comment (nemchik) [inline compose/.apps/apcupsd/apcupsd.labels.yml:16]: These should be `apcupsd_infuxdb_port` and `apcupsd_infuxdb_host`

File: compose/.apps/apcupsd/apcupsd.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+services:
+  apcupsd:
+    container_name: apcupsd
+    environment:
+      - APCUPSD_HOST=${APCUPSD_HOST}
+      - INFLUXDB_HOST=${INFLUXDB_HOST}
```
Comment (nemchik) [inline compose/.apps/apcupsd/apcupsd.yml:6]: See my comments on labels.

File: compose/.apps/apcupsd/apcupsd.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      - INFLUXDB_DATABASE=${APCUPSD_INFLUXDB_DATABASE}
+      - INFLUXDB_PASSWORD=${APCUPSD_INFLUXDB_PASSWORD}
+      - INFLUXDB_PORT=${INFLUXDB_PORT}
```
Comment (nemchik) [inline compose/.apps/apcupsd/apcupsd.yml:9]: See my comments on labels.

File: compose/.apps/apcupsd/apcupsd.labels.yml (hunk 1 above)
Comment (nemchik) [reply compose/.apps/apcupsd/apcupsd.labels.yml:16]: also p.s it's ok to pre-fill these variables with the default port for influxdb and the hostname `influxdb` as it's named in DS. Notice we do this with phpmyadmin pointing to mariadb as well.

Comment (angrycuban13) [reply compose/.apps/apcupsd/apcupsd.labels.yml]: Changed, please review.

File: compose/.apps/apcupsd/apcupsd.yml (hunk 3 above)
Comment (angrycuban13) [reply compose/.apps/apcupsd/apcupsd.yml]: Changed, please review.

File: compose/.apps/apcupsd/apcupsd.yml (hunk 2 above)
Comment (angrycuban13) [reply compose/.apps/apcupsd/apcupsd.yml]: Changed, please review.

File: compose/.apps/apcupsd/apcupsd.yml (hunk 3 above)
Comment (angrycuban13) [reply compose/.apps/apcupsd/apcupsd.yml]: Changed, please review.

**Pull request merged**